docs/pages/indexer.mdx

```diff
@@ -364,8 +372,14 @@ You can now use [`psql`](https://www.postgresql.org/docs/current/app-psql.html)
    \dt "0x5FbDB2315678afecb367f032d93F642f64180aa3__".*
    ```
 
+1. Specify that the default schema is the world being indexed.
```

Comment:
Should we say that it is <world_address>__<namespace>?
And here our tables are in the default (root) namespace hence we use the empty string. Otherwise this lgtm!